frontend/packages/dev-console/integration-tests/support/pages/functions/installOperatorOnCluster.ts

```diff
@@ -18,6 +18,12 @@ export const installOperator = (operatorName: operators) => {
       cy.get(operatorsPO.operatorHub.installingOperatorModal).should('be.visible');
       app.waitForLoad();
       cy.byTestID('success-icon').should('be.visible');
+      // For pipelines operator, tasks are not getting loaded, so introducing static wait as temporary fix
+      if (operatorName === operators.PipelinesOperator) {
+        cy.log(`Waiting for pipeline tasks to load`);
+        // eslint-disable-next-line cypress/no-unnecessary-waiting
+        cy.wait(80000);
+      }
```

Comment:
Waiting on a specific operator shouldn't be done in this generic code. Furthermore this is a very long wait time. Is there not a better way to wait on a status instead of an arbitrary amount of time?

Reply:
Its only for pipelines operator, even though it is present in generic functions and also if pipelines operator is already installed this wait time is not applicable.
Even if status is displaying as expected, still the pipelines tab doesn't display immediately.

Reply:
You should move this to a `installPipelineOperator` function instead and have a conditional wait that doesn't always add `80000ms` of time to the test run.
Waiting is an anti-pattern: https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

Reply:
Removed the wait and replaced with cy.request()